src/InterruptStepper.h

```diff
@@ -33,9 +46,10 @@ class InterruptStepper
     static uint8_t cur_ramp_level;
     static uint8_t cur_ramp_level_step;
     static uint32_t decel_start;
+    static step_callback callback;

```

Comment:
InterruptStepper should be hardware agnostic, and needs a way to tell the device to step